docs/Execution.md

```diff
@@ -1,92 +1,133 @@
-TODO: add details for how to configure a PubSub client to talk to FCA's PubSub Client.
+- **PubSub Client**: 
+A PubSub client is necessary to trigger FCA remotely.
+Users can also send messages via WebStock (WSS) on local area network (LAN).
+Before using FCA with API access, implement a PubSub client as per the [PubSub documentation](plugins/PubSub.md).
 
 ### Usage
 
-TODO: Add details for how to send a PubSub message to FCA's PubSub client
-
-FCA includes supported PubSub handlers designed to manage various incoming PubSub methods. For more information on the supported PubSub please see the [PubSub Handlers documentation](pubSubHandlers/PubSubHandlers.md).
+- Send a PubSub message to FCA's client follwoing the instructions in the [PubSub Handlers documentation](pubSubHandlers/PubSubHandlers.md).
+
+- **Sample Intent Message**:
+A sample intent message to trigger FCA remotely is provided below:
+
+```
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "APIName",
+    "params": {
+        "appId": "AppId",
+        "intent": {
+            "action": "search",
+            "data": {
+                "query": {
+                    "task": "runTest",
+                    "params": {
+                        "certification": true,
+                        "exceptionMethods": [
+                            {
+                                "method": "Api.method",
+                                "param": {
+                                    "type": "distributor"
+                                }
+                            },
+                            "..."
+                        ],
+                        "methodsToBeExcluded": [
+                            "Api.method",
+                            "Api.method",
+                            "..."
+                        ],
+                        "modulesToBeExcluded": []
+                    },
+                    "action": "CORE",
+                    "context": {
+                        "communicationMode": "SDK"
+                    },
+                    "metadata": {
+                        "target": "targetName",
+                        "targetVersion": "targetVersion",
+                        "...": "..."
+                    },
+                    "asynchronous": false,
+                    "appType": "firebolt"
+                }
+            },
+            "context": {
+                "source": "device"
+            }
+        }
+    }
+}
+
+```
+
+for more specific triggering follow instructions 
+[RunTest Handler documentation](intentReaderHandlers/RunTestHandler.md).
+
+
+# Test Types
+
+## Sanity
+
+Sanity testing ensures that all APIs function correctly at a basic level, serving as a quick check method.
+Execute the Sanity Test Suite from the UI, as described in the 'Supported Ways of Execution' section.
+This suite tests all APIs by invoking predefined calls under each API, offering a comprehensive overview of basic functionality.
+
+## Regression
```

Comment:
@omarsabha91 after discussing with Avani let's remove regression from this for now. I think we'd need more info and without being able to discuss how it works with TTT it would likely be more confusing than helpful to readers.